libs/client/client.ts

```diff
@@ -370,6 +370,29 @@ class Client {
       return null;
     }
   }
+
+  /**
+   *
+   * @function saveCommitments do the communications with commitments/saveAll
+   * endpoint
+   * @param listOfCommitments a list of commitments to be saved in the database.
+   * @author luizoamorim
+   */
+  async saveCommitments(listOfCommitments: Commitment[]) {
+    try {
+      console.log("COMMITMENTSSSSSSSSSSSSSSSSSSS:", listOfCommitments);
+      await axios.post(`${this.apiUrl}/commitment/saveAll`, listOfCommitments);
+      logger.info("Commitments imported successfully");
+    } catch (err) {
+      if (err.response.status == 500) {
+        logger.error(
+          "Some of these commitments already existis in the database!",
```

Comment:
An HTTP 500 can be related with many things, then we can't make the assumption that it was related to duplicated commitments.

Reply:
In line with isra:

- Function docstring: use `@method` instead of function, remove author 
- Remove the console.log
- Possibly new error handling will be on master, please update method accordingly
- Let's return something meaningful